### PR TITLE
21918-add-AcceptForTestCommand

### PR DESCRIPTION
**Pull request by MarcusDenker**
add AcceptForTestCommand

This adds a menu to accept a non-accepted method for test only

### CODE REVIEW

**Comment by dionisiydk** [comment]
Hi Marcus.

Build failed because of bad dependencies. We should decide how to package it correctly.
Now It depends on Commander, SUnit, Reflectivity and Reflectivity-Tools. 
Maybe we should put it into Calypso SUnit plugin.

Also name of class looks too abstract for me. What about AcceptMethodChangesForTestCommand?
And I would change variable names. Notice that commands are supposed to be usable out of tools. For example directly from playground. In that case there are no selected method or selected class.
I would use names: method and newSourceCode (instead of code). And I would remove selectedClass because it can be extracted from the method.

File: src/Reflectivity-Tools/AcceptForTestCommand.class.st [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+		arguments: #(context arguments).
+	selectedMethod ast link: metaLink.
+]
+
+{ #category : #execution }
+AcceptForTestCommand >> prepareFullExecutionInContext: aToolContext [
+	super prepareFullExecutionInContext: aToolContext.
+	selectedMethod := aToolContext selectedMethod.
+	selectedClass := selectedMethod methodClass.
+	code := aToolContext tool textModel getString.
```
Comment (dionisiydk) [inline src/Reflectivity-Tools/AcceptForTestCommand.class.st:53]: change it by "code := aToolContext tool pendingText"

**Comment by dionisiydk** [comment]
And I believe we should unify all "managed" metalinks like Breakpoint, Watchdog and ExecutionCounter. They have a lot of duplications. 
And probably this new kind of metalink should be managed same way. So it also should be managed.

And it is good opportunity to split Reflectivity-Tools to extract UI part. Because now we can't have breakpoints in image without UI (package has UI classes).

**Comment by Ducasse** [comment]
Thanks Denis. Yes thinking in terms of layered architecture is important.

**Comment by MarcusDenker** [comment]
Breakpoint, Watchdog and ExecutionCounter unification is on my TODO... but this one is different: it is not a tool related to an AST node. It just uses metalinks to add code at the start of the method (purely as an implementation strategy). It can not be applied to nodes other than method node.

**Comment by Ducasse** [comment]
So what do we do with this PR?

**Comment by MarcusDenker** [comment]
I need to redo it

**Comment by MarcusDenker** [comment]
will close it for now

**Pull request closed without merging**